werkzeug/contrib/fixers.py

```diff
@@ -142,11 +142,12 @@ def __call__(self, environ, start_response):
             'werkzeug.proxy_fix.orig_http_host':        getter('HTTP_HOST')
         })
         forwarded_for = [x for x in [x.strip() for x in forwarded_for] if x]
+        forwarded_proto = [x for x in [x.strip() for x in forwarded_proto] if x]
         remote_addr = self.get_remote_addr(forwarded_for)
         if remote_addr is not None:
             environ['REMOTE_ADDR'] = remote_addr
         if forwarded_host:
-            environ['HTTP_HOST'] = forwarded_host
+            environ['HTTP_HOST'] = forwarded_host[0]
```

Comment:
you edited the wrong line here. above you added handling for the proto but here the host is being handled.

Reply:
ahh ok it just a quick copy/paste from my reimplementation, I'll fix it later